tests/unit/runTests.ts

```diff
@@ -137,6 +137,11 @@ describe('runTests', () => {
 			assert.equal(runTests.parseArguments({ functional: true })[1], 'suites=');
 		});
 
+		it('Should add the default Reporter if none are provided', () => {
```

Comment:
While this tests the `internReporter` I am not sure about the whole `packagePath.substr(process.cwd().length + 1)` and it isn't being tested in this test, so how do I know that is working as designed?